### PR TITLE
[Silabs][EFR32] Adds fix for TC-DRLK-2.10 for the EFR32MG24 and WF200 combo

### PR DESCRIPTION
**Pull request by rosahay-silabs**
Adds code to enable MATTER shell for EFR32MG24 and WF200 combination.

### CODE REVIEW

File: examples/platform/silabs/efr32/spi_multiplex.c [hunk 1]
```diff
@@ -15,13 +15,17 @@
...
+#include "sl_wfx.h"
+#endif /* EFR32MG24 && WF200_WIFI */
+
 #include "dmadrv.h"
 #include "em_bus.h"
 #include "em_cmu.h"
 #include "em_gpio.h"
 #include "em_ldma.h"
 #include "em_usart.h"
+#include "spi_multiplex.h"
```
Comment (jmartinez-silabs) [inline examples/platform/silabs/efr32/spi_multiplex.c:28]: should be in the #ifdef above with the related wf200 stuff

Comment (rosahay-silabs) [reply examples/platform/silabs/efr32/spi_multiplex.c:28]: this was not added additionally, it was just moved from top to bottom by the restyler.

Comment (jmartinez-silabs) [reply examples/platform/silabs/efr32/spi_multiplex.c:28]: humm ok i see but spi_multiplex.h isn't used/needed for none-wifi ncp builds it should probably be in a ifdef still

Comment (rosahay-silabs) [reply examples/platform/silabs/efr32/spi_multiplex.c:29]: updated

**Review by jmartinez-silabs — APPROVED** [review]

**Review by bzbarsky-apple — APPROVED** [review]

**Pull request merged**